### PR TITLE
fixed navbar

### PR DESCRIPTION
**Pull request by fidan93**
fixed nav-sider bug on reload, cleaned up navbar component

### CODE REVIEW

File: src/components/common/NavBar.js [hunk 1]
```diff
@@ -1,22 +1,9 @@
 import React, { useContext, useState } from 'react';
 import styled from 'styled-components';
-import { Menu, Dropdown, Button } from 'antd';
-import NavMenu from './NavMenu';
+import { Button } from 'antd';
```
Comment (L-Steinmacher) [inline src/components/common/NavBar.js:4]: Nice cleanup.

File: src/components/common/NavBar.js [hunk 2]
```diff
@@ -39,7 +26,7 @@ const StyledNavBar = styled.header`
   }
   h1 {
     color: white;
-    /* margin: 0; */
```
Comment (L-Steinmacher) [inline src/components/common/NavBar.js:42]: Thank you for taking this out!

File: src/components/common/NavBar.js [hunk 3]
```diff
@@ -65,7 +53,6 @@ function NavBar(props) {
     const elem = document.querySelectorAll('.navSider');
 
     elem.forEach(el => {
-      console.log(el);
       el.classList.toggle('active');
```
Comment (L-Steinmacher) [inline src/components/common/NavBar.js:56]: Taking out console.logs!!!

**Review by L-Steinmacher — APPROVED** [review]
Thank you for making these changes!

**Review by eet349 — APPROVED** [review]
Thank you for making these changes. Good job!

**Pull request merged**